native/cocos/core/Root.cpp

```diff
@@ -292,9 +310,13 @@ void Root::frameMove(float deltaTime, int32_t totalFrames) {
     }
 
     if (_pipelineRuntime != nullptr && !_cameraList.empty()) {
-        _swapchains.clear();
-        _swapchains.emplace_back(_swapchain);
+        //_swapchains.clear();
+        //_swapchains.emplace_back(_swapchain);
         _device->acquire(_swapchains);
+
+        _cameraList[0]->changeTargetWindow(_windows[0]);
+        _cameraList[1]->changeTargetWindow(_windows[1]);
```

Comment:
Camera bind window every frame? And how to know bind which window?

Reply:
Just for testing

Reply:
No need binging every frame，the target window can be specified from Editor.

Reply:
Got it, then it is better to add comments to say `testing codes`.

Reply:
👌